openedx/core/djangoapps/programs/tests/test_utils.py

```diff
@@ -187,31 +189,57 @@ def test_get_program_for_certificates_id_not_exist(self):
         actual = get_programs_for_credentials(self.user, credential_data)
         self.assertEqual(actual, [])
 
+    def _create_enrollments(self, *course_ids):
+        return [CourseEnrollmentFactory(user=self.user, course_id=c) for c in course_ids]
```

Comment:
any chance that these will be created too close together in time to predict how they will be sorted?   (i guess we'll see)